app/main.py

```diff
@@ -1,8 +1,35 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(
+            self,
+            name: str,
+            age: int,
+    ):
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = []
+
+    for person in people:
+        Person(person["name"], person["age"])
+
+    for person in people:
+        our_object = Person.people[person["name"]]
+
+        if "wife" in person and person["wife"]:
+            our_object.wife = Person.people[person["wife"]]
+            result.append(our_object)
+
+        elif "husband" in person and person["husband"]:
+            our_object.husband = Person.people[person["husband"]]
+            result.append(our_object)
+
+        else:
+            result.append(our_object)
```

Comment:
better write `append` outside if-else statement so you don't need to write it three times